internal/kafka/internal/services/quota/quota_management_list_service.go

```diff
@@ -12,39 +14,57 @@ import (
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
 )
 
+const (
+	billingModelStandard = "standard"
+	defaultBillingModel  = billingModelStandard
+)
+
 type QuotaManagementListService struct {
 	connectionFactory   *db.ConnectionFactory
 	quotaManagementList *quota_management.QuotaManagementListConfig
 	kafkaConfig         *config.KafkaConfig
 }
 
+var _ services.QuotaService = &QuotaManagementListService{}
+
 // don't validate billing accounts when using the quota list
 func (q QuotaManagementListService) ValidateBillingAccount(organisationId string, instanceType types.KafkaInstanceType, billingCloudAccountId string, marketplace *string) *errors.ServiceError {
 	return nil
 }
 
-func (q QuotaManagementListService) CheckIfQuotaIsDefinedForInstanceType(username string, organisationId string, instanceType types.KafkaInstanceType) (bool, *errors.ServiceError) {
+func (q QuotaManagementListService) CheckIfQuotaIsDefinedForInstanceType(username string, organisationId string, instanceType types.KafkaInstanceType, kafkaBillingModel config.KafkaBillingModel) (bool, *errors.ServiceError) {
 	orgId := organisationId
+	var org quota_management.Organisation
```

Comment:
No need to predeclare the variable. Is there some reason is it done this way? Same applies to the `account` variable

Reply:
Removed.